### ISSUE TITLE
Wrong extraction of seqid in MATLAB

### ISSUE THREAD

**Issue by schuenke** [kherz/pulseq-cest-library#12]
At the beginning of the MATLAB files used to create the .seq files the sequence name / seqid is determined using 
```
if strcmp(mfilename, 'LiveEditorEvaluationHelperESectionEval')
    [~, seqid] = fileparts(matlab.desktop.editor.getActiveFilename);
else
    [~, seqid] = fileparts(which(mfilename));
end
```

When I run the code in MATLAB (R2021a),  **_mfilename_** is something like **LiveEditorEvaluationHelperESectionEval493613c8** instead of just **LiveEditorEvaluationHelperESectionEval**. To get the correct name I have to use `if contains(mfilename, 'LiveEditorEvaluationHelperESectionEval')`. Could you check if this works for your MATLAB version as well? Than I would change it in all files and create a PR.

**Comment by kherz** [comment]
that works for me, thanks for checking

**Comment by kherz** [comment]
closed with pr #14
